scripts/create_multisig_tx.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# Copyright (c) Walrus Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+GAS_OBJECT_ID=""
+TX_TYPE=""
+
+# Addresses
+WALRUS_ADMIN="0x62a69ba94e191634841cc4d196e70ec3e4667fc78013ae6d7405a0c593b39f1e"
+WALRUS_OPS="0x23eb7ccbbb4a21afea8b1256475e255b3cd84083ca79fa1f1a9435ab93d2b71b"
+
+# Object IDs
+SUBSIDIES_ADMIN_CAP="0x7cd09be8545e524217e6f35e5c306b64e3545594879dc2590e024f42bce439c6"
+SUBSIDIES_UPGRADE_CAP="0x632e10712d32b0851a1109d5a7f09680d11c74ffa5ba50eef7f14d85385cb615"
+
+
+usage() {
+  echo "Usage: $0 [OPTIONS]"
+  echo "OPTIONS:"
+  echo "  -t <tx_type> Transaction type, mandatory ('upgrade-subsidies', 'migrate-subsidies')"
+  echo "  -g <obj_id>  Gas object ID, defaults to gas object with highest balance"
+}
+
+gas_obj_for_addr() {
+  ADDRESS=$1
+  if [[ -z $GAS_OBJECT_ID ]]
+  then
+    sui client gas $ADDRESS  --json | jq -r 'max_by(.mistBalance) | .gasCoinId'
+  else
+    echo $GAS_OBJECT_ID
+  fi
+}
+
+upgrade_subsidies() {
+  GAS=$(gas_obj_for_addr $WALRUS_ADMIN)
+  GAS_BUDGET=$(sui client object $GAS --json | jq -r '.content.fields.balance')
```

Comment:
Question: AFAIU, this means that if that gas object is used for any other transaction in the meantime, this one will fail, correct? Is this intentional? If not, can we reduce the gas budget?